ShopperInsights/src/main/java/com/braintreepayments/api/ShopperInsightsClient.kt

```diff
@@ -66,18 +69,44 @@ class ShopperInsightsClient @VisibleForTesting internal constructor(
             return
         }
 
-        paymentReadyAPI.processRequest(request)
+        // TODO: get correct merchant ID from SDK
+        val merchantId = "MXSJ4F5BADVNS"
+
+        // Default values
+        val countryCode = "US"
+        val currencyCode = "USD"
+        val constraintType = "INCLUDE"
+        val paymentSources = listOf("PAYPAL", "VENMO")
+        val includeVaultTokens = true
+        val includeAccountDetails = true
```

Comment:
Do we have to send these bools?

Reply:
`includeAccountDetails` yes.
`includeVaultTokens` from testing it did not make a difference but just added it just in case.

Reply:
Gotcha - I think we should remove the ones that don't make a difference, start with as little as possible and then add if needed

Reply:
removed `includeVaultTokens` not needed.